test/e2e/gardener/shoot/create_rotate_delete.go

```diff
@@ -344,6 +345,11 @@ var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
 			})
 
 			Context("with workers rollout, in-place update strategy", Label("with-workers-rollout", "in-place"), Ordered, func() {
+				// TODO(shafeeqes, acumino, ary1992): Debug the flaky test for ipv6.
+				if os.Getenv("IPFAMILY") == "ipv6" {
+					Skip("The test is extremelly flaky for IPv6 single-stack Shoot")
```

Comment:
Nit: Not that it matters, but anyway
```suggestion
					Skip("The test is extremely flaky for IPv6 single-stack Shoot")
```